api/db-cdr/generate-cdr/generate-local-cdr-db.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+# Local or remote  mysql databases named cdr<cdr-version> and public<cdr-version> are created and populated
```

Comment:
This doesn't handle remote DBs does it? Also, if I understand correctly, this script only produces one DB -- either cdr or public -- and is called by other scripts twice to produce both... the comments should say that.

Reply:
I hate my comments :) I've changes so many. lol. I'm saving high level comments for the end from now on . Fixed.